xml/System.Text.Json/Utf8JsonWriter.xml

```diff
@@ -87,7 +82,8 @@ This is the reason why the constructor accepts a <xref:System.Text.Json.JsonWrit
         <summary>Gets the total number of bytes committed to the output by the <see cref="T:System.Text.Json.Utf8JsonWriter" /> so far for the current instance of the <see cref="T:System.Text.Json.Utf8JsonWriter" />.
             This indicates how much the <see cref="T:System.Buffers.IBufferWriter`1" /> has advanced.</summary>
         <value>The total number of bytes committed to the output by the <see cref="T:System.Text.Json.Utf8JsonWriter" /> so far.</value>
-        <remarks>To be added.</remarks>
+        <remarks>In the case of IBufferwriter, this is how much the IBufferWriter has advanced.
```

Comment:
We should not add this remark yet. The API surface for preview3 does not have stream support.

Reply:
When was this added @ahsonkhan?

Reply:
Removed.

Reply:
> When was this added @ahsonkhan?

It was added as part of the type re-design in preview5 (see https://github.com/dotnet/corefx/pull/36961). A few of the docs got updated because of the change in the API shape/type (for instance it is no longer a ref struct).

See the change in the API-diff here: https://github.com/dotnet/core/pull/2638/files#diff-bfc4229a5bee882bfb7f7c687a1286efR26